src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -125,6 +125,40 @@ class ReviewCollapsibleChapter extends React.Component {
     return pageTitle;
   };
 
+  getPageDetails = page => {
+    const { form } = this.props;
+    const pageState = form.pages[page.pageKey];
+
+    if (page.showPagePerItem) {
+      return {
+        pageSchema:
+          pageState.schema.properties[page.arrayPath].items[page.index],
+        pageUiSchema: pageState.uiSchema[page.arrayPath].items,
+        pageData: get([page.arrayPath, page.index], form.data),
+        arrayFields: [],
+        fullPageKey: `${page.pageKey}${page.index}`,
+      };
+    }
+    // TODO: support array fields inside of an array page?
+    // Our pattern is to separate out array fields (growable tables) from
+    // the normal page and display them separately. The review version of
+    // ObjectField will hide them in the main section.
+
+    // This will be undefined if there are no fields other than an array
+    // in a page, in which case we won’t render the form, just the array
+    const pageSchemaObjects = getNonArraySchema(
+      pageState.schema || {},
+      pageState.uiSchema || {},
```

Comment:
If I'm reading things correctly, we didn't have this `|| {}` check before. Can you speak a bit on this change? One concern that comes to my mind is whether other code might depend on pageState.schema and .uiSchema being an object or null/undefined.

Reply:
Besides this change, I don't see any other substantive changes to this method. But if there are any other substantive changes that I'm missing, can you call them out?

Reply:
Ahhh, you're right. I accidentally left this check in while trying to troubleshoot the `findArrays` error. I've ended up removing the `getPageDetails` function and calculating the schema & uiSchema using the same method into the `getCustomPageContent` function.